arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Iterable.kt

```diff
@@ -844,6 +844,10 @@ public fun <A> Iterable<A>.salign(
  * <!--- KNIT example-iterable-11.kt -->
  * <!--- TEST lines.isEmpty() -->
  */
+@Deprecated(
+  "Unzip is being deprecated in favor of the standard library version.\n$NicheAPI",
+  ReplaceWith("unzip()", "kotlin.collections.unzip")
+)
 public fun <A, B> Iterable<Pair<A, B>>.unzip(): Pair<List<A>, List<B>> =
   fold(emptyList<A>() to emptyList()) { (l, r), x ->
```

Comment:
This should just delegate to stdlib `unzip`

Reply:
Yes, that's exactly what I intended! I think an import alias is just the right approach here. I wonder how the `ReplaceWith` will function because of the name discrepancy though. `kotlin.collections.*` is a default import, and so perhaps we could just deprecate this with `HIDDEN` level. I'd rather wait though on another reviewer's opinion as to whether `HIDDEN` is the right approach here or not.